src/runtime/rpc/rpc_host_port.cpp

```diff
@@ -113,4 +142,43 @@ void host_port::assign_group(const char *name)
     _group_host_port->add_ref();
 }
 
+error_s host_port::resolve_addresses(std::vector<rpc_address> &addresses) const
+{
+    CHECK(addresses.empty(), "invalid addresses, not empty");
```

Comment:
```suggestion
    CHECK(addresses.empty(), "");
```
Since the CHECK will print the content "addresses.empty()", so it's not needed to print the message again, except it can show more extra information.